src/nvim/ex_cmds_defs.h

```diff
@@ -126,39 +126,36 @@ struct exarg {
   struct condstack *cstack;     ///< condition stack for ":if" etc.
 };
 
-#define FORCE_BIN 1             /* ":edit ++bin file" */
-#define FORCE_NOBIN 2           /* ":edit ++nobin file" */
-
-/* Values for "flags" */
-#define EXFLAG_LIST     0x01    /* 'l': list */
-#define EXFLAG_NR       0x02    /* '#': number */
-#define EXFLAG_PRINT    0x04    /* 'p': print */
-
-/*
- * used for completion on the command line
- */
-typedef struct expand {
-  int xp_context;                       /* type of expansion */
-  char_u      *xp_pattern;              /* start of item to expand */
-  int xp_pattern_len;                   /* bytes in xp_pattern before cursor */
-  char_u      *xp_arg;                  /* completion function */
-  int xp_scriptID;                      /* SID for completion function */
-  int xp_backslash;                     /* one of the XP_BS_ values */
+#define FORCE_BIN 1             // ":edit ++bin file"
+#define FORCE_NOBIN 2           // ":edit ++nobin file"
+
+// Values for "flags"
+#define EXFLAG_LIST     0x01    // 'l': list
+#define EXFLAG_NR       0x02    // '#': number
+#define EXFLAG_PRINT    0x04    // 'p': print
+
+// used for completion on the command line
+struct expand {
+  int xp_context;      // type of expansion
+  char_u *xp_pattern;  // start of item to expand
+  int xp_pattern_len;  // bytes in xp_pattern before cursor
+  char_u *xp_arg;      // completion function
+  int xp_scriptID;     // SID for completion function
+  int xp_backslash;    // one of the XP_BS_ values
```

Comment:
just a small suggestion: typically it's good to leave some extra whitespace in case a new, longer-named member is added here, to avoid having to re-align all of the comments. E.g., these comments could be aligned with line 135 above.